services/movie_session.py

```diff
@@ -42,3 +42,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider adding error handling for cases where a `MovieSession` might not be found. Use `MovieSession.objects.get(id=session_id).delete()` within a try-except block to catch `MovieSession.DoesNotExist` exceptions.